src/DependencyInjection/Configuration.php

```diff
@@ -584,7 +584,6 @@ public function getConfigTreeBuilder(): TreeBuilder
             ->end();
 
         $this->addGelfSection($handlerNode);
-        $this->addMongoSection($handlerNode);
```

Comment:
The [comments](https://github.com/symfony/monolog-bundle/blob/27d779a384e4da1e4132e7a427414bbc1b0a8217/src/DependencyInjection/Configuration.php#L82) on the top should be removed as well.